build/testing/test.go

```diff
@@ -79,7 +79,7 @@ func WithSQLite(ctx context.Context, client *dagger.Client, container *dagger.Co
 
 func WithPostgres(ctx context.Context, client *dagger.Client, flipt *dagger.Container) (context.Context, *dagger.Client, *dagger.Container) {
 	return ctx, client, flipt.
-		WithEnvVariable("FLIPT_TEST_DB_URL", "postgres://postgres:password@postgres:5432").
+		WithEnvVariable("FLIPT_TEST_DB_URL", "postgres://postgres:password@postgres:5432?sslmode=disable").
```

Comment:
im honestly not sure how this works without disabling SSL for unit tests with PG. It seems we disable SSL for cockroachdb already for our unit tests so I would think we would need to do the same for PG?

Reply:
Does our test suite not do this for us. Or perhaps it uses one of the modes by default that lets the server tell the client what to do (`allow`, `prefer` etc.)